atlasdb-jdbc/build.gradle

```diff
@@ -9,6 +9,8 @@ dependencies {
 
   compile "io.dropwizard:dropwizard-jackson:" + libVersions.dropwizard
   compile "org.jooq:jooq:3.6.4"
+  compile 'org.postgresql:postgresql:9.4.1208'
```

Comment:
Not sure you want this as compile dependency (I.e. Folks depending on AtlasDB-jdbc shouldn't pick this up transatively), I think you want this as test runtime provided.